extension.json

```diff
@@ -37,6 +48,18 @@
 	"config": {
 	},
 
+	"Rights": {
+		"can-approve-pages": {
+			"description": "Allows users to approve pages",
+			"groups": []
+		}
```

Comment:
Didn't we decide that this right is not necessary? If a user has been linked to a category then that implies the user has permission to approve.

Reply:
Having a right to control who can manage approvers makes sense, but I agree that there seems to be no benefit of having a right for controlling who can approve.

Reply:
@waximabbax I corrected the task you created: https://github.com/ProfessionalWiki/PageApprovals/issues/31